guidelines/index.html

```diff
@@ -1942,11 +1942,11 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
             
             <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/compatible.html">Understanding Compatible</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#compatible">How to Meet Compatible</a></div><p>現在及び将来の、支援技術を含むユーザエージェントとの互換性を最大化すること。</p>
             
-            <section class="guideline" id="parsing"><div class="header-wrapper"><h4 id="x4-1-1-parsing-obsolete-and-removed"><bdi class="secno">Success Criterion 4.1.1 </bdi>Parsing (Obsolete and removed)</h4><a class="self-link" href="#parsing" aria-label="Permalink for Section 4.1.1"></a></div>
+            <section class="guideline" id="parsing"><div class="header-wrapper"><h4 id="x4-1-1-parsing-obsolete-and-removed"><bdi class="secno">達成基準 4.1.1 </bdi>構文解析 (廃止・削除)</h4><a class="self-link" href="#parsing" aria-label="Permalink for Section 4.1.1"></a></div>
    
    
 
-   <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/parsing.html">Understanding Parsing (Obsolete and removed)</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#parsing">How to Meet Parsing (Obsolete and removed)</a></div><div class="note" role="note" id="issue-container-generatedID-43"><div role="heading" class="note-title marker" id="h-note-43" aria-level="5"><span>注記</span></div><p class="">This criterion was originally adopted to address problems that assistive technology had directly parsing HTML. Assistive technology no longer has any need to directly parse HTML. Consequently, these problems either no longer exist or are addressed by other criteria. This criterion no longer has utility and is removed.</p></div>
+   <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/parsing.html">Understanding Parsing (Obsolete and removed)</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#parsing">How to Meet Parsing (Obsolete and removed)</a></div><div class="note" role="note" id="issue-container-generatedID-43"><div role="heading" class="note-title marker" id="h-note-43" aria-level="5"><span>注記</span></div><p class="">この達成基準はもともと、支援技術が直接 HTML を構文解析する際の問題に対処するために採用された。支援技術は、もはや直接 HTML を構文解析する必要がなくなった。その結果、この問題は存在しない、又は他の達成基準によって対処される。この達成基準はもはや有用ではないため、削除された。</p></div>
```

Comment:
原文には no longer が3回登場してくどいなぁと思うわけですが、訳文には「もはや」が2回しか登場していないようです。落ちているのはこの部分ですね。

>Consequently, these problems either no longer exist or are addressed by other criteria.
>その結果、この問題は存在しない、又は他の達成基準によって対処される。

原文がくどいのは意図的なものだと思いますので、ここの no longerはくどく訳出した方が良いと思います。あと、

- these なので「この」→「これらの」ですかね。
- are addressed を「対処される」としていますが、これだと現時点で対処されているのかどうか分かりにくいように思いました。すでに他の達成基準でカバーできているという話だと思いますので、 「対処されている」くらいのほうがしっくりくるように思います。

といったところで以下のようにしてはどうでしょうか?

```suggestion
   <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/parsing.html">Understanding Parsing (Obsolete and removed)</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#parsing">How to Meet Parsing (Obsolete and removed)</a></div><div class="note" role="note" id="issue-container-generatedID-43"><div role="heading" class="note-title marker" id="h-note-43" aria-level="5"><span>注記</span></div><p class="">この達成基準はもともと、支援技術が直接 HTML を構文解析する際の問題に対処するために採用された。支援技術は、もはや直接 HTML を構文解析する必要がなくなった。その結果、これらの問題はもはや存在しないか、又は他の達成基準によって対処されている。この達成基準はもはや有用ではないため、削除された。</p></div>
```

Reply:
> these なので「この」→「これらの」ですかね。

英語で複数系になっていますが、

> This criterion was originally adopted to address **problems** that assistive technology had directly parsing HTML.
> [...] Consequently, *these problems*

この達成基準はもともと、支援技術が直接 HTML を構文解析する際の**問題**に対処するために採用された。
（中略）この問題は、

と日本語では単数になるのが自然であって、複数にする必要はないかと思いますが、どうでしょうか。

ほかはご指摘のとおりかと思います。

ところで、either A or Bを明示的に訳してみましたがどうでしょうか。

```suggestion
   <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/parsing.html">Understanding Parsing (Obsolete and removed)</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#parsing">How to Meet Parsing (Obsolete and removed)</a></div><div class="note" role="note" id="issue-container-generatedID-43"><div role="heading" class="note-title marker" id="h-note-43" aria-level="5"><span>注記</span></div><p class="">この達成基準はもともと、支援技術が直接 HTML を構文解析する際の問題に対処するために採用された。支援技術は、もはや直接 HTML を構文解析する必要がなくなった。その結果、この問題はもはや存在しないか、他の達成基準によって対処されているかのいずれかである。この達成基準はもはや有用ではないため、削除された。</p></div>
```

Reply:
>この問題はもはや存在しないか、他の達成基準によって対処されているかのいずれかである

ここは、問題が複数あることがわからないと意味がわかりにくいです。問題が単数だと思って読むと「いずれかである」と言われて「どっちやねん」という感想になります。実際には、複数の問題が存在していて、それらの中には問題がなくなっているもの、問題は残っているが他の達成基準でカバーされているものがある、という話です。

最初に登場する「問題」は、特に複数形であることがわかるように訳す必要はないと思います (複数形であるとわかるように、「諸問題」とするのもアリだと思います)。いずれにしても、あとの「この問題」のほうは「これらの問題」としたほうがわかりやすいかなと……。

```suggestion
   <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/parsing.html">Understanding Parsing (Obsolete and removed)</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#parsing">How to Meet Parsing (Obsolete and removed)</a></div><div class="note" role="note" id="issue-container-generatedID-43"><div role="heading" class="note-title marker" id="h-note-43" aria-level="5"><span>注記</span></div><p class="">この達成基準はもともと、支援技術が直接 HTML を構文解析する際の諸問題に対処するために採用された。支援技術は、もはや直接 HTML を構文解析する必要がなくなった。その結果、これらの問題は存在しない、又は他の達成基準によって対処される。この達成基準はもはや有用ではないため、削除された。</p></div>
```

Reply:
@bakera 
ありがとうございます。理解しました。
https://github.com/waic/wcag22/pull/33/commits/6e470d5fe0b3e561880252d44f94e0ebc4ceee71 にて反映させています。